<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>636</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CustomGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuMesh">
    <property name="title">
     <string>Mesh</string>
    </property>
    <addaction name="actionVertices"/>
    <addaction name="actionDraw_Line_tool"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionRender_Vertices_only"/>
    <addaction name="actionRender_Vertices_and_Edges"/>
    <addaction name="actionRender_Partial"/>
    <addaction name="actionRender_Full"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMesh"/>
   <addaction name="menuHelp"/>
   <addaction name="menuRendering"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMove"/>
   <addaction name="separator"/>
   <addaction name="actionVertices"/>
   <addaction name="actionDraw_Line_tool"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDraw_Line_tool">
   <property name="text">
    <string>Line tool</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHow_to">
   <property name="text">
    <string>How-to</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionRender_Vertices_only">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertices only</string>
   </property>
  </action>
  <action name="actionRender_Full">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full render (3 steps)</string>
   </property>
  </action>
  <action name="actionRender_Partial">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Partial render (1 step)</string>
   </property>
  </action>
  <action name="actionRender_Vertices_and_Edges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertices and Edges</string>
   </property>
  </action>
  <action name="actionVertices">
   <property name="text">
    <string>Vertices</string>
   </property>
   <property name="toolTip">
    <string>Add Vertices</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GMView/customgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
